### DISCUSSION TITLE
Allow to ignore Amazon Q inline suggestions

### DISCUSSION THREAD

**Discussion by lorenzwalthert** [aws/aws-toolkit-vscode#4957] [Ideas]
I like that Amazon Q gives me hints how to improve my code. However, I'd like to deactivate these somehow, either inline, per file, per rule or globally. I could not find docs on this. Is there a way to do that currently? Putting `# noqa` on that line does not work.
![Screenshot 2024-05-08 at 13 53 50](https://github.com/aws/aws-toolkit-vscode/assets/10477073/d015d2b2-d79c-4ad5-b90e-47fb783a5c6d)

**Comment by justinmk3** [comment]
I created a feature request for this: https://github.com/aws/aws-toolkit-vscode/issues/4958

**Comment by andrewyuq** [comment]
Hi! Thanks for the feedback.
Right now you can:

1. Click on the Amazon Q status bar to open the quickpick menu.
2. If the Auto-Scans line says `RUNNING`, click it once so that it shows `PAUSED` like below.
3. Restart VSCode and these popups will go away after the restart.

<img width="627" alt="Screenshot 2024-05-08 at 11 03 37 AM" src="https://github.com/aws/aws-toolkit-vscode/assets/89420755/8117374b-9d21-43e6-b552-d1738963e5e2">

Meanwhile we are working on a UX fix to make it less disruptive.
